test/prism_regression/local_variable_operator_write.rb

```diff
@@ -0,0 +1,17 @@
+# typed: false
+
+a += 1
+
+b -= 2
+
+c ||= 3
+
+d &&= 4
```

Comment:
These aren't `operator_write`. I suggest we either:

1. Section them off into their own commented section
2. Rename this file, e.g. `local_variable_writes.rb`
3. Extract these to `local_variable_and_write.rb` and `local_variable_or_write.rb`